### PR TITLE
change initial example everywhere

### PR DESCRIPTION
**Pull request by villares**
Action on https://github.com/berinhard/pyp5js/issues/163

### CODE REVIEW

File: docs/examples/sketch_000/sketch_000.py [hunk 1]
```diff
@@ -3,11 +3,9 @@
...
-
 
 def draw():
-    fill("blue")
     background(200)
-    radius = sin(frameCount / 60) * 50 + 50
-    ellipse(100, 100, radius, radius)
+    diameter = sin(frameCount / 60) * 50 + 50
+    fill("blue")
+    ellipse(100, 100, diameter, diameter)
```
Comment (berinhard) [inline docs/examples/sketch_000/sketch_000.py:11]: Shouldn't we replace this `ellipse` call by a `circle` one instead?

**Review by berinhard — APPROVED** [review]
This is great! Please ignore my comment on the `circle` call.

File: docs/examples/sketch_000/sketch_000.py (hunk 1 above)
Comment (villares) [reply docs/examples/sketch_000/sketch_000.py:11]: hmm, yes, we could, but I guess ellipse is alright.

**Comment by villares** [comment]
Hi Bernardo, I must have made something wrong, these changes are not reflecting on the main readme and o the demo linked by the main readme... please advise!

**Comment by berinhard** [comment]
This wasn't merged into the main branch yet, so that's the reason. I still have to get back to this.

**Pull request merged**